misc/python/genapi.py

```diff
@@ -40,6 +40,11 @@
     #'mesh send',
 ]
 
+# API Version:
+VERSION = '2.0a1'
+# minimega version (set this before rendering)
+MM_VERSION = 'UNKNOWN'
```

Comment:
We could, but then the version produced in the API file when run with
different versions of the minimega binary would be incorrect. This way you
can always run the latest genapi.py with whatever version of minimega you
want bindings for.
On Jun 5, 2015 11:12 AM, "David Fritz" <notifications@github.com> wrote:

> In misc/python/genapi.py
> <https://github.com/sandia-minimega/minimega/pull/230#discussion_r31820882>
> :
>
> > @@ -40,6 +40,11 @@
> >      #'mesh send',
> >  ]
> >
> > +# API Version:
> > +VERSION = '2.0a1'
> > +# minimega version (set this before rendering)
> > +MM_VERSION = 'UNKNOWN'
>
> See https://github.com/sandia-minimega/minimega/blob/master/build.bash#L8
>
> We auto-generate the version as the commit hash, and hand-tool a version
> number on releases. Could we do that here as well?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sandia-minimega/minimega/pull/230/files#r31820882>.
>